libsolidity/codegen/CompilerContext.cpp

```diff
@@ -52,7 +54,6 @@
 // Change to "define" to output all intermediate code
 #undef SOL_OUTPUT_ASM
 #ifdef SOL_OUTPUT_ASM
-#include <libyul/AsmPrinter.h>
 #endif
```

Comment:
Is this empty actually now? Then the ``#ifdef`` should just be removed, shouldn't it?